python/__init__.py

```diff
@@ -7,8 +7,10 @@
     pass
 
 
-from tk_framework_adobe import adobe_bridge
+from .tk_framework_adobe import adobe_bridge
```

Comment:
I'm not really sure how this was working before, since this folder was not added to the sys path. But in Python3 it now requires us to put a `.` in front as we are importing from within the package.
This reoccurs elsewhere in this PR.

Reply:
right I wondered if it was something like that from just observing the behavior. I prefer the py3 behavior I think.